dss/api/search.py

```diff
@@ -70,15 +70,12 @@ def post(json_request_body: dict,
 
         result_list = []  # type: typing.List[dict]
         for hit in page['hits']['hits']:
-            result = {}
+            result = {'bundle_id': hit['_id'],
+                      'bundle_url': _build_bundle_url(hit, replica),
+                      'search_score': hit['_score']
+                      }
             if format == 'raw':
                 result.update({'meta_data': hit['_source']})
```

Comment:
Would just do:

`result['meta_data'] = hit['_source']` as it is faster and simpler.

Reply:
will fix